Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:20.04
```

Comment:
I think we should use any one of the docker images from [here](https://hub.docker.com/r/nvidia/cuda) because some codes are assuming that we can use torch on CUDA. FYI, basically people use `xxx-devel-xxx`.

[Additional reference here](https://stackoverflow.com/questions/65492490/how-to-install-cuda-enabled-pytorch-in-a-docker-container)

Reply:
It's fine to use this ubuntu container for the dockerfile that we upload to the docker hub. I think the most general use case is to use CPU. If someone would like to use it with CUDA, it is quite easy for them to edit this file.

Reply:
Btw, i don't think pynisher works properly when we use CUDA, but have you ever tried any runs on CUDA?

Reply:
Actually thats a good point. I'll create an issue and a subsequent PR where we can use docker to run all our tests on cuda. What do you think?

Reply:
Sounds great:)